bundles/org.eclipse.e4.ui.workbench.addons.swt/src/org/eclipse/e4/ui/workbench/addons/cleanupaddon/CleanupAddon.java

```diff
@@ -15,6 +15,7 @@
 
 package org.eclipse.e4.ui.workbench.addons.cleanupaddon;
 
+import jakarta.annotation.PostConstruct;
```

Comment:
import javax.annotation.PostConstruct;
import javax.inject.Inject;

Reply:
The Eclipse SDK migrated from the javax namespace to the jakarta.
See https://eclipse.dev/eclipse/news/4.30/platform.php#support-jakarta-annotations